docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -0,0 +1,253 @@
+---
+assignees:
+- caesarxuchao
+- erictune
+
+---
+
+Ubuntu 16.04 introduced the [Canonical Distribution of Kubernetes](https://jujucharms.com/canonical-kubernetes/), a pure upstream distribution of Kubernetes designed for production usage. Out of the box it comes with the following components on 12 machines:
+
+- Kubernetes (automated deployment, operations, and scaling)
+     - Three node Kubernetes cluster with one master and two worker nodes.
+     - TLS used for communication between nodes for security.
+     - Flannel Software Defined Network (SDN) plugin
+     - A load balancer for HA kubernetes-master (Experimental)
+     - Optional Ingress Controller (on worker)
+     - Optional Dashboard addon (on master) including Heapster for cluster monitoring
+- EasyRSA
+     - Performs the role of a certificate authority serving self signed certificates
+       to the requesting units of the cluster.
+- Etcd (distributed key value store)
+     - Three node cluster for reliability.
+- Elastic stack
+     - Two nodes for ElasticSearch
```

Comment:
Should we scrub the term 'nodes' in favor of 'units'? We're mixing/matching both in this list.